src/client.cpp

```diff
@@ -109,6 +109,7 @@ ADDON_STATUS ADDON_Create(void* callbacks, void* props)
     }
 
     // Environment must be initialized before calling retro_init()
+    FRAME_TIME_LAST = timer.microseconds();
```

Comment:
This is better, because now the first frame time will only be a second or two, but shouldn't it actually be zero?

Reply:
Perhaps do an if statement to reset it? What would be the best solution do you think?